README.md

```diff
@@ -23,4 +23,7 @@ Currently built on Moltin but PRs for other systems are welcome
 - rename the `/doc/.env.template` to `/.env` and fill out the details
 
 ## More information
-- check the [Philosophy document](https://github.com/KickAssCommerce/KickAssCommerce/blob/develop/doc/philosophy.md)
\ No newline at end of file
+- check the [Philosophy document](https://github.com/KickAssCommerce/KickAssCommerce/blob/develop/doc/philosophy.md)
+
+## Examples
+Anything in the `app` directory is example code. The `app` directory should be used for your project code.
```

Comment:
Yeah... should probably be in it's own repo. Admittedly I needed an copy-pasted example in there since I had absolutely no idea what I was doing